GOSSIP.md

```diff
@@ -257,26 +265,27 @@ are selected from the secret and other discriminators.
 
 List of constants with their current default values:
 
-- Maximum number of outbound connections: 10
-- Soft maximum number of inbound connections: 100
-- Number of buckets in the verified pool: 2^8 (256)
-- Number of peer per verified buckets: 2^5 (32)
-- Number of buckets in the unverified pool: 2^10 (1024)
-- Number of peer per unverified buckets: 2^6 (64)
-- Maximum number of duplicated peers in the unverified pool: 8
-- Probability of adding a Nth duplicated reference of an existing peer to the unverified pool: '1/2^N'
+- Maximum number of outbound connections: `10`
+- Soft maximum number of inbound connections: `100`
+- Number of buckets in the verified pool: `2^8` (`256`)
+- Number of peer per verified buckets: `2^5` (`32`)
+- Number of buckets in the unverified pool: `2^10` (`1024`)
+- Number of peer per unverified buckets: `2^6` (`64`)
+- Maximum number of duplicated peers in the unverified pool: `8`
+- Probability of adding a Nth duplicated reference of an existing peer to the
+unverified pool: `1/2^N`
 - Period of verified peer random peer check when the maximum number
-of verified connections has been reached: 60 seconds
+of verified connections has been reached: `60 seconds`
 - Period of new peer connection up to the maximum number of verified
-connections: 20 seconds.
-- Gossip ping frequency: 120 seconds
-- Maximum time to get a ping from an inbound connection: 30 seconds
+connections: `min(30, 2^OUTBOUND_CONNECTION_COUNT) seconds`.
```

Comment:
I Forgot to update this it is `min(30, 2^(OUTBOUND_CONNECTION_COUNT - 1))`. I'll change it next week.